app/Http/Controllers/FishRecordController.php

```diff
@@ -53,6 +53,34 @@ public function edit(Request $request, $id)
         $rec = FishRecord::find($id);
         return view('Fish_Record_edit', ['rec' => $rec]);
     }
+    public function update(FishRecordRequest $request)
+    {   
+        
+        $Fish_Records=FishRecord::find($request->id);
```

Comment:
変数は小文字で始め、適度にスペースを加えると見やすくなるかなと思いました！
```suggestion
        $fish_records = FishRecord::find($request->id);
```